contracts/mockup/MockLockedSov.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity 0.5.17;
+pragma experimental ABIEncoderV2;
+
+import "../core/State.sol";
+
+contract LockedSovMockup is State {
```

Comment:
contract and file names if there is one contract should be the same
suggested to rename file and respective references

Reply:
This file is no longer used, i will remove